bool start
string map_name
string database_path
string linkpoint_db_path
int8 map_id
string pointcloud_dir
float32 loop_closure_search_radius
bool no_graph_connection
bool indoor
